manifests/init.pp

```diff
@@ -59,7 +59,7 @@
   String                     $package_ensure           = installed,
   String                     $package_name             = 'gitlab-runner',
   Stdlib::HTTPUrl            $repo_base_url            = 'https://packages.gitlab.com',
-  Stdlib::Fqdn               $repo_keyserver           = 'keys.gnupg.net',
```

Comment:
We shouldn't change the default value unless it is broken. This would be a breaking change.

Reply:
I agree. But the next release already contains some backwards incompatible changes

Reply:
As mentioned in the commit message, I've got several errors (about 70-90% of the tests) which can be traced back to the unreliability of the `keys.gnupg.net` keyserver.

I wouldn't count this as a breaking change because it won't change existing installation. This is because the APT key got already retrieved and won't be re-downloaded because the keyserver changed.